scripts/spack/packages/umpire/package.py

```diff
@@ -183,6 +189,16 @@ def initconfig_hardware_entries(self):
                 arch_str = ",".join(archs)
                 entries.append(cmake_cache_string(
                     "HIP_HIPCC_FLAGS", '--amdgpu-target={0}'.format(arch_str)))
+            # BEGIN AXOM EDIT
+            # Fix blt_hip getting HIP_CLANG_INCLUDE_PATH-NOTFOUND bad include directory
+            if self.spec.satisfies('%clang'):
```

Comment:
Whoops, I missed this. This should be guarded by the system type as done for raja.